packages/grafana-ui/src/components/Layout/Stack/Stack.tsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 
-import { ThemeSpacingTokens } from '@grafana/data';
+import { Flex, FlexProps } from '../Flex/Flex';
 
-import { Flex } from '../Flex/Flex';
-import { ResponsiveProp } from '../utils/responsiveness';
-interface StackProps extends Omit<React.HTMLAttributes<HTMLElement>, 'className' | 'style'> {
-  direction?: ResponsiveProp<'column' | 'row'>;
-  gap?: ResponsiveProp<ThemeSpacingTokens>;
-}
+interface StackProps extends Pick<FlexProps, 'gap' | 'direction' | 'alignItems' | 'justifyContent'> {}
```

Comment:
Personally, I would like these to not just be "flex properties". The point of Stack is to abstract away the flexbox implementation details, and `<Stack justifyContent='flex-start'/>` violates that principle.

I think we should add an `align` property to Stack and HorizontalStack, which accepts values like `start`, `center`, and `end` (or left,center,right / top,center/bottom).